README.md

```diff
@@ -10,6 +10,10 @@ The Presto Technical Charter is located in [CHARTER.md](CHARTER.md)
 
 https://lists.prestodb.io/g/presto-tsc
 
+## Public Calendar
+
+https://calendar.google.com/calendar/embed?src=linuxfoundation.org_vrjlva5b0u73ps75fvnv5sasi4%40group.calendar.google.com&ctz=America%2FChicago
```

Comment:
It may be cleaner to link to the calendar instead of putting in the raw link to the readme, something like the following. Please feel free to update the text in any way you want.

```suggestion
[Here](https://calendar.google.com/calendar/embed?src=linuxfoundation.org_vrjlva5b0u73ps75fvnv5sasi4%40group.calendar.google.com&ctz=America%2FChicago) is the Google calendar for the TSC meetings.
```